presence-detector.py

```diff
@@ -96,6 +96,10 @@ def polling_loop(self):
                 if process.returncode == 0:
                     clients = json.loads(process.stdout)
                     for client in clients['clients']:
+                        # Add ap prefix if ap_name defined in settings
+                        if bool(self.settings.ap_name):
```

Comment:
Can you adjust this as follows? I like to use f"" for readability. Also, you test the ap_name variable directly as both None or "" evaluate to False.
```suggestion
                        if self.settings.ap_name:
                            client = f"{self.settings.ap_name}_{client}"
```